app/models/mixins/miq_web_server_worker_mixin.rb

```diff
@@ -15,7 +16,7 @@ def self.binding_address
 
     def self.rails_server_command_line
       @rails_server_command_line ||= begin
-        "#{self.nice_prefix} #{Rails.root.join("bin", "rails")} server".freeze
+        "#{nice_prefix} #{Rails.root.join("bin", "rails")} server #{RAILS_SERVER}".freeze
```

Comment:
I'm asking why we need the extra parameter at all.

ISTM the only thing better than a reduced number of references to puma, would be zero (outside the Gemfile) -- leave it up to `rails server` to just use whatever it can find.